games/minimal/game.conf

```
@@ -1,2 +1,2 @@
 name = Minimal development test
-
+description = Testing environment to help with testing the core features of Minetest. It can also be helpful in mod development.
```

Engine is more understandable than core, and core usually refers to the c++ of the engine rather than its entirety

```suggestion
description = Testing environment to help with testing the engine features of Minetest. It can also be helpful in mod development.
```